libs/domain/content/src/models/content.model.ts

```diff
@@ -1,10 +1,22 @@
+declare global {
+  interface DynamicContentFields {
```

Comment:
Why `Dynamic`? Isn't all dynamic? I tend to drop prefixes like these (remember `SemanticLinkService`?)